biggerquery/user_commons/sensor.py

```diff
@@ -0,0 +1,24 @@
+import biggerquery as bgq
+
+
+YESTERDAY_WHERE_CLAUSE = "DATE(%(partitioning_column)s) = DATE(TIMESTAMP_ADD(TIMESTAMP('{dt} UTC'), INTERVAL -24 HOUR))"
```

Comment:
It is used only in tests. Maybe it will be better to move it to tests?

Reply:
done